documentation/book/assembly-kafka-bridge-quickstart.adoc

```diff
@@ -0,0 +1,57 @@
+// This assembly is included in the following assemblies:
+//
+// assembly-using-the-kafka-bridge.adoc
+
+// Save the context of the assembly that is including this one.
+// This is necessary for including assemblies in assemblies.
+// See also the complementary step on the last line of this file.
+:parent-context-using-the-kafka-bridge: {context}
+:context: {parent-context-using-the-kafka-bridge}-bridge-quickstart
+
+[id='assembly-kafka-bridge-quickstart-{context}']
+= Kafka Bridge quick start
+
+Use this quick start to try out the {ProductName} Kafka Bridge in your local development environment. You will learn how to:
+
+* Deploy the Kafka Bridge to your OpenShift cluster
+* Expose the Kafka Bridge Service to your local machine
+* Produce messages to topics, including to a specific partition
+* Create a Kafka Bridge consumer for use in this quick start
+* Perform basic consumer operations
```

Comment:
is "for use in this quick start" needed? It's clear that we are going to use the consumer for the quick start, or?
I guess that this point could be merged with the last one as:

* Create a Kafka Bridge consumer and perform basic consumer operations

Reply:
Good suggestion, thank you.